packages/imperative/src/constants/src/Constants.ts

```diff
@@ -59,6 +59,8 @@ export class Constants {
     public static readonly JSON_OPTION_ALIAS = "rfj";
     public static readonly HELP_OPTION = "help";
     public static readonly HELP_OPTION_ALIAS = "h";
+    public static readonly VERSION_OPTION: "version";
+    public static readonly VERSION_OPTION_ALIAS: "v";
```

Comment:
The alias for version is a capital `V` which is case sensitive
```suggestion
    public static readonly VERSION_OPTION_ALIAS: "V";
```